src/plugins/gui/providers/ioniaProvider.ts

```diff
@@ -223,6 +228,8 @@ export const makeIoniaProvider: FiatProviderFactory<IoniaMethods> = {
     // State:
     //
 
+    let hiddenCardIds: number[] = asStoreHiddenCards(await store.getItem(STORE_HIDDEN_CARDS_KEY).catch(_ => undefined))
+    console.log('!@!', { hiddenCardIds })
```

Comment:
Could be useful for debugging to have these logged but with a more useful message

Reply:
Not intended to be there.